src/Platforms/AbstractPlatform.php

```diff
@@ -73,6 +73,10 @@ abstract class AbstractPlatform
 
     public const CREATE_FOREIGNKEYS = 2;
 
+    private $types = [
+        Types\BooleanType::class => PlatformTypes\BooleanType::class
+    ];
```

Comment:
Why is this map needed? it's never written to, shouldn't one class name be built from the other class name?